src/main/java/gregtech/common/metatileentities/multi/MetaTileEntityLargeBoiler.java

```diff
@@ -313,7 +309,7 @@ private int setupRecipeAndConsumeInputs() {
                     fluidTank.drain(fuelAmountToConsume, true);
                     long recipeVoltage = FuelRecipeLogic.getTieredVoltage(dieselRecipe.getMinVoltage());
                     int voltageMultiplier = (int) Math.max(1L, recipeVoltage / GTValues.V[GTValues.LV]);
-                    return (int) Math.ceil(dieselRecipe.getDuration() * CONSUMPTION_MULTIPLIER / 2.0 * voltageMultiplier * getThrottleMultiplier());
```

Comment:
Why are you removing the throttle multiplier from this value, but not from when the fuel amount to consume is calculated, which is a couple lines above?

Reply:
If I remember well that's the main part of the bug the throttle was applied too many times thus acting more like a secondary efficiency than like a throttle